src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,15 @@
-const fizzBuzz = number => {};
+const fizzBuzz = number => {
+if(number % 5 === 0 && number % 3 === 0) {
+    return "FizzBuzz"
+}
+    if (number % 5 === 0){
+        return "Buzz"
+    }
+if (number % 3 === 0) {
+    return "Fizz"}
+
+    return number
+
```

Comment:
Careful with indentation.
Prettier might be your best friend here (like the one Mo has and showed in class).